src/OpenTelemetry.Exporter.Console/ConsoleExporterOptions.cs

```diff
@@ -0,0 +1,29 @@
+// <copyright file="ConsoleExporterOptions.cs" company="OpenTelemetry Authors">
+// Copyright 2018, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Exporter.Console
+{
+    /// <summary>
+    /// Options for the <see cref="ConsoleTraceExporter" />.
+    /// </summary>
+    public class ConsoleExporterOptions
+    {
+        /// <summary>
+        /// Gets or sets a value indicating whether output should be pretty printed.
+        /// </summary>
+        public bool PrettyPrint { get; set; }

```

Comment:
Would the use case of this exporter be for pure visualization on console apps?
If so, would it be better to default this to true?

```suggestion
        public bool PrettyPrint { get; set; } = true;
```